README.md

```diff
@@ -62,15 +64,20 @@ Within model/config/experiment, the experimental_configs dictionary can be modif
         }
     }
 
-Where each key represents a set, the config_option_state refers to a specific starting state, config_option_params does the same for parameters, monte_carlo_n defines the number of monte carlo runs, and T defines the number of timesteps for the experiment.
+Here 
+* each key represents a set that can be run,
+* the `config_option_state` refers to a specific starting state,
+* ` config_option_params` does the same for parameters,
+* `monte_carlo_n` defines the number of monte carlo runs, and
+* `T` specifies the number of timesteps for the experiment.
 
-Within model/config/params, the config_option_map is set up to handle the different options for creating parameter sweeps. More detail in the parameters section on specifics.
+Within `model`/`config`/`params`, the `config_option_map` is set up to handle the different options for creating parameter sweeps. More detail in the parameters section on specifics.
```

Comment:
I'm not sure how I feel about using the quotations here for file paths. I think it makes sense for some of the params and variables but file paths are one value. And by this I mean model/config/params is a path to code